examples/Protobuf-legacy/cmake/Hunter/config.cmake

```diff
@@ -0,0 +1,14 @@
+if(ANDROID OR IOS)
+  hunter_config(Protobuf
+      VERSION ${HUNTER_Protobuf_VERSION}
+      CMAKE_ARGS
+      protobuf_BUILD_PROTOC=OFF
+      protobuf_BUILD_LIBPROTOBUF_LITE=OFF
```

Comment:
We can set this by default by the way: https://github.com/ruslo/hunter/wiki/usr.adding.new.package#defaults